src/cloudant/account.py

```diff
@@ -404,25 +399,24 @@ def _usage_endpoint(self, endpoint, year=None, month=None):
 
     def bill(self, year=None, month=None):
         """
-        _bill_
-
-        Get your cloudant billing data, optionally for a given year/month
+        Retrieves Cloudant billing data, optionally for a given year/month.
 
-        :param year: int, year, eg 2014
-        :param month: int, 1-12
+        :param int year: Year to query against, eg 2014.  Defaults to None
+        :param int month: Month to query against, eg 1-12.  Defaults to None
 
-        :returns: JSON billing data structure
+        :returns: Billing data in JSON format
```

Comment:
The main purpose for adding the "in JSON format" throughout was because of the way we wrap a lot of responses in a `Result` object in this library.  I wanted to call out to the user that they should expect JSON when we return it and a `Result` when we return a `Result`.  It would be a good bit of work to update the docs with links.  Plus adding the links will make the docstrings look even worse than they look now when someone would choose to view them in a command window which was the original purpose of docstrings.  Anyway, it is something to think about but likely not for this PR.